calm-visualizer/src/components/graph/Graph.tsx

```diff
@@ -2,94 +2,123 @@ import { useEffect } from "react";
 import './Graph.css'
 import { BrowserJsPlumbInstance } from "@jsplumb/browser-ui"
 import { NodeLayout, RelationshipLayout } from "../../layout";
-import { CALMConnectsRelationship, CALMInteractsRelationship, CALMRelationship } from "../../types";
+import { CALMComposedOfRelationship, CALMConnectsRelationship, CALMDeployedInRelationship, CALMInteractsRelationship, CALMRelationship } from "../../types";
 
 interface Props {
     instance: BrowserJsPlumbInstance,
     nodes: NodeLayout[],
     relationships: RelationshipLayout[]
 }
 
-// const groups: string[] = [];
-
 function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationship: CALMConnectsRelationship) {
-    const r = relationship["relationship-type"]["connects"];
-    instance.connect({
-        source: document.getElementById(r.source.node)!,
-        target: document.getElementById(r.destination.node)!,
-        anchor: "Continuous",
-        connector: {
-            type:"Straight",
-            options: {
-                "stub": 25
-            }
-        },
-        endpoint: {
-            type: "Blank",
-            options: {}
-        },
-        overlays: [
-            { type:"Arrow", options:{location:1}}
-        ]
-    });
+                const r  = relationship["relationship-type"]["connects"];
```

Comment:
why has so much whitespace been added here? It seems a bit excessive to me

Reply:
Resolved